CMSgov/easi-app
# [EASI-2942] Fix components

# EASI-2942

## Changes and Description

- Fix calculation of TRB requester component
- Remove TRB Lead component

## How to test this change

- `scripts/dev up`
- `scripts/dev db:seed`
- Ensure TRB Lead info shows correctly on Admin home view
- Ensure that requester component shows properly

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
